common/spec/dependabot/dependency_group_spec.rb

```diff
@@ -117,17 +117,17 @@
       context "when no dependencies are assigned to the group" do
         it "returns true if the dependency matches a pattern" do
           expect(dependency_group.dependencies).to eq([])
-          expect(dependency_group.contains?(test_dependency1)).to be_truthy
+          expect(dependency_group.contains?(test_dependency1)).to be(true)
```

Comment:
This changes the expectation from looking for a truthy value to checking for exactly `true`. This is not what I was suggesting; I suggested aliasing the `contains?` method to `include?` so that we can write this test in the form that Jamie suggested

Reply:
well, the predicate matcher rubocop rule actually suggested the changes.  **.contains?** is a method on the dependency_group object.  I guess we could rewrite that method to only respond with true or false, and never nil and get the same effect.  No matter what, the "to be_truthy" and "to be_falsey" will fail this cop.


https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/PredicateMatcher

Reply:
That's not what I'm suggesting. Maybe read what I wrote again carefully